pypeit/core/wavecal/spectrographs/templ_gemini_gmos.py

```diff
@@ -0,0 +1,107 @@
+""" Generate the wavelength templates for Keck/DEIMOS"""
```

Comment:
"Generate the wavelength templates for Gemini/GMOS"

Reply:
fixed!